wavelength_router/wavelength_router.proto

```diff
@@ -36,12 +36,38 @@ service WavelengthRouter {
   // updates this way. There is no means by which an adjustment that has been
   // triggered by a client can be cancelled unless the CancelAdjustPSD RPC is
   // used.
-  rpc AdjustPSD(AdjustPSDRequest) returns (stream AdjustPSDResponse) {}
+  // DEPRECATED, use AdjustSpectrum.
+  rpc AdjustPSD(AdjustPSDRequest) returns (stream AdjustPSDResponse) {
+    option deprecated = true;
+  }
 
   // CancelAdjustPSD cancels an in-progress AdjustPSD request. If an
   // adjustment is not being performed for the provided request, then the RPC
   // should return successfully.
-  rpc CancelAdjustPSD(AdjustPSDRequest) returns (CancelAdjustPSDResponse) {}
+  // DEPRECATED, use CancelAdjustSpectrum.
+  rpc CancelAdjustPSD(AdjustPSDRequest) returns (CancelAdjustPSDResponse) {
+    option deprecated = true;
+  }
+
+  // AdjustSpectrum performs a spectrum power adjustment on an optical
+  // DWDM line system port.
+  //
+  // This RPC's initial message triggers adjustments. The target should validate
+  // the request, and immediately begin the adjustment as long as another
+  // adjustment is not already running. As this RPC may take multiple minutes,
+  // the state should be regularly streamed to the client. The operation should
+  // be completed on the device regardless of the state of the RPC channel to
+  // the initiating client. A client that becomes disconnected during an
+  // operation can reconnect, requesting the same operation, and receive status
+  // updates this way. There is no means by which an adjustment that has been
+  // triggered by a client can be cancelled unless the CancelAdjustSpectrum
+  // RPC is used.
+  rpc AdjustSpectrum(AdjustSpectrumRequest) returns (stream AdjustSpectrumResponse) {}
+
+  // CancelAdjustSpectrum cancels an in-progress AdjustSpectrum request. If an
+  // adjustment is not being performed for the provided request, then the RPC
+  // should return successfully.
+  rpc CancelAdjustSpectrum(AdjustSpectrumRequest) returns (CancelAdjustSpectrumResponse) {}
 }
 
 message AdjustPSDRequest {
```

Comment:
Do we need to mark these messages deprecated as well ?  What is the proto best practice?

Reply:
I learned something new that a message can be marked deprecated :)